internal/apiserver/route_post_new_subscription.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 func newSubscriptionSchemaGenerator(ctx context.Context) string {
-	baseSchema, _, _ := openapi3gen.NewSchemaRefForValue(&fftypes.Subscription{})
+	baseSchema, _ := openapi3gen.NewSchemaRefForValue(&fftypes.Subscription{}, nil)
```

Comment:
Is there any issue with passing nil here? @peterbroadhurst

Reply:
Think that's fine - would only be a problem if we ever introduce a recursive schema into one of the subscription types 👍